lib/honeycomb-rails/subscribers/active_record.rb

```diff
@@ -20,7 +20,11 @@ def call(*args)
         data = event.payload.slice(:name, :connection_id)
         data[:sql] = event.payload[:sql].strip
         event.payload[:binds].each do |b|
-          data["bind_#{ b.name }".to_sym] = b.value
+          if b.is_a?(Array)
+            data["bind_#{ b[0].name }".to_sym] = b[1]
+          elsif b.is_a?(Object)
+            data["bind_#{ b.name }".to_sym] = b.value
+          end
```

Comment:
Style nits:

 * more idiomatic to use a `case` statement:
    ```ruby
    case b
    when Array
      # foo
    else
      # bar
    end
    ```
 * `b.is_a?(Object)` will always evaluate to `true`, even if `b` is `nil`, so this is probably not testing for what you think it is testing for

Reply:
LOL. It's been a long time since I've written idiomatic Ruby :) thanks.